src/map/battle.cpp

```diff
@@ -5003,7 +5007,8 @@ static int battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list *
 			break;
 		case SU_LUNATICCARROTBEAT:
 		case SU_LUNATICCARROTBEAT2:
-			skillratio += 100 + 100 * skill_lv;
+			skillratio += 100 + 100 * skill_lv + sstatus->str / 6; // !TODO: What's the STR bonus?
```

Comment:
Status x 5 is a magic number that they alway use. Maybe it better to use this formula?